docs/en/transform-v2/dynamic-compile.md

```diff
@@ -88,7 +88,7 @@ transform {
     compile_pattern="SOURCE_CODE"
     source_code="""
                  import org.apache.seatunnel.api.table.catalog.Column
-                 import org.apache.seatunnel.transform.common.SeaTunnelRowAccessor
+                 import org.apache.seatunnel.api.table.type.SeaTunnelRowAccessor
```

Comment:
Can we keep the old `SeaTunnelRowAccessor` class too? To make sure the old dynamic config can work well too.